src/Tizen.WebView/Interop/Interop.ChromiumEwk.Context.cs

```diff
@@ -37,5 +37,28 @@ public enum CacheModel
 
         [DllImport(Libraries.ChromiumEwk)]
         internal static extern CacheModel ewk_context_cache_model_get(IntPtr context);
+
+        [DllImport(Libraries.ChromiumEwk)]
+        internal static extern void ewk_context_resource_cache_clear(IntPtr context);
+
+        [DllImport(Libraries.ChromiumEwk)]
+        internal static extern void ewk_context_notify_low_memory(IntPtr context);
+
+        internal delegate void DownloadStartCallback(IntPtr url, IntPtr userData);
+
+        [DllImport(Libraries.ChromiumEwk)]
+        internal static extern void ewk_context_did_start_download_callback_set(IntPtr context, DownloadStartCallback callback, IntPtr userData);
+
+        [DllImport(Libraries.ChromiumEwk)]
+        internal static extern int ewk_context_menu_item_tag_get(IntPtr contextMenuItem);
```

Comment:
enum ContextMenuItemTag is also needed for interop.
Plz refer to CacheModel.

Reply:
Yeah I saw CacheModel,
will it not be overhead ?? such a big enum in 2 files, thats why i did type casting..